AWS/legos/aws_get_ec2_instances_with_smaller_cpu_size/aws_get_ec2_instances_with_smaller_cpu_size.py

```diff
@@ -0,0 +1,116 @@
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+from typing import List, Optional
+from pydantic import BaseModel, Field
+import pprint
+from unskript.legos.aws.aws_list_all_regions.aws_list_all_regions import aws_list_all_regions
+from unskript.legos.aws.aws_get_all_ec2_instances.aws_get_all_ec2_instances import aws_get_all_ec2_instances
+
+
+class InputSchema(BaseModel):
+    region: Optional[str] = Field(
+        '', description='AWS Region to get the RDS Instance', title='AWS Region'
+    )
+    instance_ids: Optional[List] = Field(
+        '', description='List of instance IDs to check.', title='List of Instance IDs'
+    )
+    threshold: Optional[float] = Field(
+        default= 2,
+        description='The CPU size threshold. Default value is 2.0. Size map is as follows-\n    "nano": 2,\n    "micro": 2,\n    "small": 1,\n    "medium": 1,\n    "large": 2,\n    "xlarge": 4,\n    "2xlarge": 8,\n    "3xlarge": 12,\n    "4xlarge": 16,\n    "6xlarge": 24,\n    "8xlarge": 32,\n    "9xlarge": 36,\n    "10xlarge": 40,\n    "12xlarge": 48,\n    "16xlarge": 64,\n    "18xlarge": 72,\n    "24xlarge": 96,\n    "32xlarge": 128,\n    "metal": 96',
```

Comment:
To improve code readability its better if we can write this as a multi line string using `"""`